### ISSUE TITLE
Problem using one click installer under Windows

### ISSUE THREAD

**Issue by SoftologyPro** [XmYx/ainodes-engine#27]
1. Create a new directory
2. Download the latest one click installer from https://github.com/XmYx/ainodes-engine/releases/tag/v0.4.2 and save into the new directory
3. Run the one click installer.
4. Once it finishes double-click run_ainodes.bat

The logo flashes for less than a second and then nothing.  GUI no longer starts.

No command line stats.
```
(nodes_env) D:\ainodes\ainodes-engine>run_ainodes.bat

(nodes_env) D:\ainodes\ainodes-engine>
```

**Comment by XmYx** [comment]
Hey!

I've pushed a fix this morning, please try running update_ainodes.bat, if the app does not start up, run debug_venv.bat, and enter ```python main.py``` then report here with the full error stack from the command prompt please.

Thanks,
Miki

**Comment by SoftologyPro** [comment]
OK, running update_ainodes.bat had these errors during running

```
INFO: pip is looking at multiple versions of tensorflow to determine which version is compatible with other requirements. This could take a while.
ERROR: Cannot install -r requirements.txt (line 11), -r requirements.txt (line 4) and protobuf==3.20.3 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested protobuf==3.20.3
    tensorboard 2.8.0 depends on protobuf>=3.6.0
    tensorflow 2.8.4 depends on protobuf<3.20 and >=3.9.2

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts

[notice] A new release of pip is available: 23.1.2 -> 23.2.1
[notice] To update, run: python.exe -m pip install --upgrade pip
Processing ainodes_engine_comfy_nodes
warning: redirecting to https://github.com/XmYx/ainodes_engine_comfy_nodes.git/
Already up to date.
ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'

and when it finished the same logo flicker for half a second then nothing.

Running the debug did give more info

```
D:\ainodes\ainodes-engine>debug_venv.bat

(nodes_env) D:\ainodes\ainodes-engine>python main.py
Start aiNodes, please wait. 2023-07-24 21:03:01.549169
Using HF Cache in app dir
Importing from ai_nodes\ainodes_engine_base_nodes
ai_nodes.ainodes_engine_base_nodes.audio_generation_nodes.audiocraft_music_node
===== 2023.07.24 21:07:03 =====
Traceback (most recent call last):
  File "D:\ainodes\ainodes-engine\main.py", line 126, in <module>
    import_nodes_from_subdirectories(folder_path)
  File "D:\ainodes\ainodes-engine\ainodes_frontend\base\import_utils.py", line 54, in import_nodes_from_subdirectories
    import_nodes_from_directory(subdir_path)
  File "D:\ainodes\ainodes-engine\ainodes_frontend\base\import_utils.py", line 43, in import_nodes_from_directory
    module = importlib.import_module(f"{dir}.{module_name}")
  File "importlib\__init__.py", line 126, in import_module
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "D:\ainodes\ainodes-engine\ai_nodes\ainodes_engine_base_nodes\audio_generation_nodes\audiocraft_music_node.py", line 9, in <module>
    from audiocraft.modules.conditioners import ConditioningAttributes
ModuleNotFoundError: No module named 'audiocraft'

(nodes_env) D:\ainodes\ainodes-engine>
```

**Comment by XmYx** [comment]
Cool cool, this helps. I'll get back to you with a fix.

**Comment by jags111** [comment]
I am having a similar problem and now trying to fix same by reinstalling all requirements one by one. every time we run the update bat it keeps downloading everything from start and the takes almost a long time even though the files exists. May be the windows permission issues are at the core and also file not being located to the version of python in the system. Not sure what the issue is but it is a lot of work to solve the install issues.

**Comment by XmYx** [comment]
Hi!

I'm working on a major update, which improved both the venv and how nodes are executed, I suggest that you do the following for a new installation, for exiting ones, delete the nodes_env, and skip first step.:

`git clone https://github.com/XmYx/ainodes-engine
cd ainodes-engine
git fetch
git checkout dev
setup_ainodes.bat`

make sure it downloads ComfyUI into the src directory, and that the ai_nodes/ainodes_engine_base_nodes folder is in the dev branch. To do that, just navigate to the folder in command line, and:

`git fetch
git checkout dev
git pull`

Basically switched to a miniconda environment, and added an auto node runner, so pressing End on the keyboard runs the nodes placed, if they can (all required inputs connected). There is a keybind editor in the settings, and a small flaw that makes it not save the diretories, but if you manually edit the settings.yaml, you can set the dirs, i'll work on fixing saving from gui, if you wish to change any current keybind, or model directory.

**Comment by jags111** [comment]
Hey it worked perfect with your new instructions and all good and as you mentioned the same; settings.yaml in the config directory needs editing to update paths.  rest i am exploring and will keep you posted and looks all good now in my windows, 3096 configuration and testing the render speeds and deforum both and will update in the deforum or other discord servers. Good work ! a quick question on what are the color palette for and may be settings needs to be explained in the basics section for any one wanting the learn the UI. You can start a git hub -- info page for same and keep adding info there !

**Comment by kyle215ps3** [comment]
I tried all of this but it's not working and comfyui is not in the src folder...

**Comment by XmYx** [comment]
Solved. Sorry to have left the issue stale.
